### ISSUE TITLE
[WCCMMUD] MajorMUD - Text overflows line boundaries

### ISSUE THREAD

**Issue by paladine** [mbbsemu/MBBSEmu#267]
**Module Information**
* Identifier [WCCMMUD]
* Module Name MajorMud
* Version Doesn't matter

**Describe the bug**
Words are split up across lines, meaning part of a word can be on one line and the rest on the next line.

**To Reproduce**
Enter Mud, just look at room descriptions. It's quite self evident.

**Expected behavior**
Lines break at 80 character boundaries, like legacy MBBS/WG.
*However*, the best implementation here would read the telnet window size that the client sends. The client will inform the server how many characters per line it can display, so to cap at that number would be ideal.

**Comment by paladine** [comment]
I've got similar code in my TA reimplementation that does this.

**Comment by enusbaum** [comment]
One thing to keep in mind is we'll need to strip the ANSI control sequences off the output when counting characters. I have similar code in the FsdRoutine when calculating X/Y coordinates of the FSD Input Fields.

**Comment by enusbaum** [comment]
In MajorBBS/Worldgroup, the values for the users terminal are stored on `usracc->scnwid` for X (width), and `usracc->scnbrk` for Y (height)

**Comment by tuday2** [comment]
This is also in Galgsbl
```
        /// <summary>
        ///     Sets Screen Width (ignored)
        /// </summary>
        private void btutsw()
        {
            var channel = GetParameter(0);
            var width = GetParameter(1);

#if DEBUG
            _logger.Warn($"Set Screen Width for Channel {channel} to {width}");
#endif
            Registers.AX = 0;
        }
```

**Comment by enusbaum** [comment]
A complexity here, is you'll need to inspect the output buffer to see if it includes any ANSI cursor command sequences. You could have 80+ characters in the buffer, but they're to draw ANSI menus on the screen, etc.

We need to be careful not to inject `\r\n` on more complex ANSI sequences (BladeMaster, etc.) if we misidentify it as a long string.

**Comment by fletcherm** [comment]
@paladine @enusbaum Any of the above relevant given the recent changes, or can this be closed?

**Comment by enusbaum** [comment]
Correct, this can be closed with #521 being merged in.
